mhs/outbound/fix_pycurl.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+pipenv run pip install --no-cache-dir --compile --ignore-installed --install-option="--with-openssl" --install-option="--openssl-dir=/usr/local/opt/openssl" pycurl
```

Comment:
Can you give more info what it does?

Reply:
That wasn't supposed to be here :) This script fixes `ImportError: pycurl: libcurl link-time ssl backend (openssl) is different from compile-time ssl backend (none/other)` - it was easier for me to just run script instead of copying it from readme file. That can help others so we can leave it here or I'll remove it.